### PR TITLE
chore: Bump package versions to address security vulnerabilities

### PR DESCRIPTION
**Pull request by pevey**
This PR bumps the package versions on the following packages in @medusajs/medusa:

class-validator
express
jsonwebtoken
multer
passport
uuid

I believe this will close https://github.com/medusajs/medusa/issues/1024  as it should close all the critical security issues generated by `npm audit`.

I haven't noticed any issues yet while using the new package versions.

3 tests fail, but I think they are only failing because the specific error code that is expected is not the error code now generated.

```
Summary of all failing tests
 FAIL  src/api/routes/admin/price-lists/__tests__/create-price-list.ts (208.3s)
  ● POST /price-lists › fails if required fields are missing › returns descriptive error that several fields are missing

    expect(received).toEqual(expected) // deep equality

    Expected: "name must be a string, type must be a valid enum value, prices must be an array"
    Received: "name must be a string, type must be one of the following values: sale, override, prices must be an array"

      112 |     it("returns descriptive error that several fields are missing", () => {
      113 |       expect(subject.body.type).toEqual("invalid_data")
    > 114 |       expect(subject.body.message).toEqual(
          |                                    ^
      115 |         "name must be a string, type must be a valid enum value, prices must be an array"
      116 |       )
      117 |     })

      at Object.<anonymous> (src/api/routes/admin/price-lists/__tests__/create-price-list.ts:114:36)

 FAIL  src/api/routes/admin/orders/__tests__/create-claim.js (208.356s)
  ● POST /admin/orders/:id/claims › fails to create a claim when type is not known › throws an error

    expect(received).toEqual(expected) // deep equality

    Expected: "type must be a valid enum value"
    Received: "type must be one of the following values: refund, replace"

      82 |     it("throws an error", () => {
      83 |       expect(subject.status).toEqual(400)
    > 84 |       expect(subject.body.message).toEqual("type must be a valid enum value")
         |                                    ^
      85 |     })
      86 |   })
      87 |

      at Object.<anonymous> (src/api/routes/admin/orders/__tests__/create-claim.js:84:36)

  ● POST /admin/orders/:id/claims › successfully creates a claim with a reason › fails to create a claim when type is not known › throws an error

    expect(received).toEqual(expected) // deep equality

    Expected: "reason must be a valid enum value"
    Received: "reason must be one of the following values: missing_item, wrong_item, production_failure, other"

      166 |       it("throws an error", () => {
      167 |         expect(subject.status).toEqual(400)
    > 168 |         expect(subject.body.message).toEqual(
          |                                      ^
      169 |           "reason must be a valid enum value"
      170 |         )
      171 |       })

      at Object.<anonymous> (src/api/routes/admin/orders/__tests__/create-claim.js:168:38)


Test Suites: 2 failed, 200 passed, 202 total
Tests:       3 failed, 1174 passed, 1177 total
Snapshots:   4 passed, 4 total
Time:        216.99s
Ran all test suites.
```

### CODE REVIEW

**Comment by pevey** [comment]
It looks like the GitHub actions are blocking the CI tests because of the following:

"The lockfile would have been modified by this install, which is explicitly forbidden."

**Comment by olivermrbl** [comment]
@pevey Appreciate this contribution. I'll do a couple of modifications to see if I can make the CI play nice - hope its cool :)

**Comment by pevey** [comment]
The tests finish on my machine and pass (except the ones mentioned in the PR), but the test script does not seem to exit gracefully.  I'm not sure the cause.  Googling leads to these 2 reported issues, not much help.

https://github.com/vercel/turbo/issues/1648
https://github.com/vercel/turbo/issues/1683

Here is the ending output on my machine when running yarn test -vvv.  (WSL Ubuntu 22.04.2 x86-64, nodejs v18.16.0)
```
@medusajs/medusa:test: Test Suites: 2 failed, 200 passed, 202 total
@medusajs/medusa:test: Tests:       3 failed, 1174 passed, 1177 total
@medusajs/medusa:test: Snapshots:   4 passed, 4 total
@medusajs/medusa:test: Time:        10.682s, estimated 116s
@medusajs/medusa:test: Ran all test suites.
2023-04-15T06:14:38.580-0500 [ERROR] turbo.: Error: command finished with error: command (/home/lacey/dev/medusa/packages/medusa) yarn run test exited (1)
@medusajs/medusa:test: ERROR: command finished with error: command (/home/lacey/dev/medusa/packages/medusa) yarn run test exited (1)
2023-04-15T06:14:38.580-0500 [DEBUG] turbo.processes.(/home/lacey/dev/medusa/packages/medusa-payment-paypal) yarn run test: Kill(%v) called: EXTRA_VALUE_AT_END=false
2023-04-15T06:14:38.580-0500 [DEBUG] turbo.processes.(/home/lacey/dev/medusa/packages/oas/medusa-oas-cli) yarn run test: Kill(%v) called: EXTRA_VALUE_AT_END=false
2023-04-15T06:14:38.580-0500 [DEBUG] turbo.processes.(/home/lacey/dev/medusa/packages/medusa-payment-stripe) yarn run test: Kill(%v) called: EXTRA_VALUE_AT_END=false
command (/home/lacey/dev/medusa/packages/medusa) yarn run test exited (1)
```

**Comment by pevey** [comment]
Just to add to the above comment, if you go into these packages mentioned in the error individually and run `yarn test`, the tests all pass:

medusa-payment-paypal
medusa-oas-cli
medusa-payment-stripe

So, I'm pretty baffled.

**Comment by olivermrbl** [comment]
@pevey tests are failing because type checking on enum values was changed - for the better - in [this PR](https://github.com/typestack/class-validator/pull/1826). I will correct the tests.

**Comment by pevey** [comment]
Whatever I just did, I did not mean to do.  I was trying to pull your changes to my local machine.

File: integration-tests/api/__tests__/admin/currency.js [hunk 1]
```diff
@@ -54,39 +54,6 @@ describe("/admin/currencies", () => {
       expect(response.data).toMatchSnapshot()
     })
   })
-
-  describe("POST /admin/currencies/:code", function () {
```
Comment (olivermrbl) [inline integration-tests/api/__tests__/admin/currency.js:58]: comment: As of 0.14.0, the `class-validator` forbids unknown values by default. This test is therefore redundant, as the `includes_tax` property will not be allowed payload when the FF is disabled.

Comment (olivermrbl) [reply integration-tests/api/__tests__/admin/currency.js:58]: https://github.com/typestack/class-validator/blob/develop/CHANGELOG.md#breaking-changes

**Review by olivermrbl — APPROVED** [review]
LGTM - thanks @pevey 💯

**Pull request merged**